web/client-api/src/main/java/io/deephaven/web/public/table_basic.html

```diff
@@ -0,0 +1,71 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <title>Deephaven JS API table viewport and updates</title>
+    <script src="dh-internal.js" type="text/javascript"></script>
+    <script src="dh-core.js" type="text/javascript"></script>
+    <link href="basic.css" rel="stylesheet" type="text/css" />
+    <style>
+        table { border-spacing: 0; }
+        thead td { font-weight: bold; border-bottom: 1px solid black; }
+        tbody td { border-bottom: 1px solid gray; padding: 4px; }
+    </style>
+</head>
+<body>
+
+<h3>Table data</h3>
+<table id="simpleTable" />
+
+<script>
+(async () => {
+    // Open a connection to the server
+    var connection = new dh.IdeConnection(window.location.protocol + "//" + window.location.host);
+
+    // Start a python session
+    var ide = await connection.startSession("python");
+
+    // Run code that will create a static table with 10 rows and three columns, I, J, K
+    await ide.runCode(`
+from deephaven.TableTools import emptyTable
+remoteTable = emptyTable(10).updateView("I=i", "J=I*I", "K=i%2==0?\`Hello\`:\`World\`")
```

Comment:
i hate the escaping quoting in quoting - python supports single quotes, maybe use that instead? ` -> ' -> "